src/Gui/Stylesheets/Dark-blue.qss

```diff
@@ -64,7 +64,7 @@ KNOWN BUGS and TO DO
 /*==================================================================================================
 Reset elements
 ==================================================================================================*/
-/* Reseting everything helps to unify styles across different operating systems */
+/* Resetting everything helps to unify styles across different operating systems */

```

Comment:
@pgilfernandez FYI you've re-introduced the typos we fixed previously for the .qss files.  Please pull these changes in to your branch to avoid reintroducing them again. Thanks.

Reply:
doh...
Done, thanks